### ISSUE TITLE
Import SCORME 1.2 made with iSpring

### ISSUE THREAD

**Issue by ivlevi** [chamilo/chamilo-lms#2558]
### Current behavior / Resultado actual / Résultat actuel
Error uploading SCORME course (zip file) to Learning Paths
### Expected behavior / Resultado esperado / Résultat attendu
Should upload the material and listed in course learning path

### Steps to reproduce / Pasos para reproducir / Étapes pour reproduire
Create course. Go to Learning Path. Click upload SCORME. Select zip file. Click upload

### Chamilo Version / Versión de Chamilo / Version de Chamilo
1.11.6

**Comment by ywarnier** [comment]
Hi @ivlevi, could you share the SCORM package you are having issues with? This is usually an error in the package, so we have to check it.
You don't mention which error message you get when you upload. This can help us give you a solution.

**Comment by ivlevi** [comment]
Sure! Thank you for swift reply!
Question first: the file (zip) is 35 mb. How you prefer to do this?

And:
I took a look here:
![image](https://user-images.githubusercontent.com/40044266/41203549-6a568e3c-6ce1-11e8-93d3-79bf948d9118.png)

and here

![image](https://user-images.githubusercontent.com/40044266/41203555-74dd43f0-6ce1-11e8-8c87-380f864d27d2.png)

Sugestions?

**Comment by ywarnier** [comment]
Yup, post_max_size *and* upload_max_filesize have to be at least 10% more than the file you are trying to upload. So if your file is 32MB and those two values are 8MB, that won't work.

In the install guide, section 2 > PHP Configuration, you'll find that we recommend setting those to 100MB straight. Maybe reading some of the documentation we write with love would have helped?
https://campus.chamilo.org/documentation/installation_guide.html#2._Installation_of_Chamilo_LMS

**Comment by ivlevi** [comment]
I have modified the upload limit to 64 mb. Still not uploading...

**Comment by ywarnier** [comment]
Both?

**Comment by ivlevi** [comment]
I did this:
![image](https://user-images.githubusercontent.com/40044266/41203660-e86ac59e-6ce2-11e8-9413-f0d6de8b7150.png)

Where can I find these:
short_open_tag       = Off
    safe_mode            = Off
    magic_quotes_gpc     = Off
    magic_quotes_runtime = Off

**Comment by ivlevi** [comment]
My MAN!
It work the upload! And believe'it or not, I can even see the damn thing!

Where should I send the beer?!

**Comment by ivlevi** [comment]
:)
Thank but...
I'm in Romania, so...
But if you plan a vacation in Europe, don't waste your time in Paris or alike: come here! I'll introduce you to prince Charles on his monthly visit here!
:)

PS: I can use a one to one lesson to use and set Chamilo> Can you recomend anyone? Just reading the manual is not enough for me (*dumb).

Best!

**Comment by ywarnier** [comment]
There are different online courses available for learning Chamilo (and a few more in preparation). I cannot really recommend anyone at the moment because we don't usually do one-to-one lessons, but you could ask on the forum who's willing to do that.
The Chamiluda events are free and decentralized: people just coordinate with a school for a free classroom on a Saturday and share their (usually limited, but always welcome) knowledge about Chamilo.
That brings me to a question: how is the Romanian translation of Chamilo? Good? I hired someone to add the translation about 3 years ago, and I don't know Romanian, so I'm curious to know.
